### PR TITLE
feature/9-registration-page-mobile

### LINKED ISSUE #9: Registration Page Mobile Screen
**Link to User Story**
[1007](https://github.com/d-lama/.github/blob/main/mgmt/product_backlog.md#1007)

**Feature Description**
Create Registration Page View

- [ ] Create Page that checks if the device is mobile
- [ ] Create Component for mobile view of registration
- [ ] Create condition in component if registration falied or succeeded

### PR DESCRIPTION
**Pull request by vonwareb**
- created Registration.tsx and Registration.css
- added necessary inputfields
- added "Register" button

closes #9

### CODE REVIEW

File: src/components/forms/Button.css
Comment (schazadm) [inline src/components/forms/Button.css:1]: it's alright to remove or not add .css files if not used

File: src/pages/Registration.css
Comment (schazadm) [inline src/pages/Registration.css:1]: here the same as Button.css

File: src/components/forms/Input.css
Comment (schazadm) [inline src/components/forms/Input.css:1]: same as Button.css

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                            {/*                            <IonButton size="large" className="ion-text-center" expand="block" shape="round"
+                                       onClick={() =>
+                                           presentAlert({
+                                               header: 'Attention!',
+                                               message: 'Registration page not yet implemented!',
+                                               buttons: ['OK'],
+                                           })
+                                       }
+                            >Sign Up
+                            </IonButton>*/}
```
Comment (schazadm) [inline src/pages/Login.tsx:64]: remove dead code

File: src/pages/Registration.tsx
Comment (schazadm) [inline src/pages/Registration.tsx:1]: looks pretty good 😁👍

**Review by schazadm — CHANGES_REQUESTED** [review]
overall formatting the code would be good for readability
otherwise already very well 👍

File: src/pages/Registration.css
Comment (vonwareb) [reply src/pages/Registration.css:1]: removed Registration.css

File: src/components/forms/Input.css
Comment (vonwareb) [reply src/components/forms/Input.css:1]: removed Input.css

File: src/App.tsx
[the remark this answers is not in the document]
Comment (vonwareb) [reply src/App.tsx:1]: Done

File: src/pages/Login.tsx (hunk 1 above)
Comment (vonwareb) [reply src/pages/Login.tsx:64]: Done

File: src/pages/Registration.tsx
Comment (vonwareb) [reply src/pages/Registration.tsx:1]: Thank you for the feedback :-)

**Pull request merged**